shipment_management/public/js/get_rates.js

```diff
@@ -1,92 +1,109 @@
 frappe.ui.form.on("Quotation", {
-    refresh: function (frm) {
-        add_fedex_rates_button(frm);
-    },
+	refresh: function(frm) {
+		add_fedex_rates_button(frm);
+	},
 
-    get_fedex_rates: function (frm) {
-        _get_fedex_rates(frm);
-    }
+	get_fedex_rates: function(frm) {
+		_get_fedex_rates(frm);
+	}
 });
 
 frappe.ui.form.on("Sales Order", {
-    refresh: function (frm) {
-        add_fedex_rates_button(frm);
-    },
+	refresh: function(frm) {
+		add_fedex_rates_button(frm);
+	},
 
-    get_fedex_rates: function (frm) {
-        _get_fedex_rates(frm);
-    }
+	get_fedex_rates: function(frm) {
+		_get_fedex_rates(frm);
+	}
 });
 
 frappe.ui.form.on("Sales Invoice", {
-    refresh: function (frm) {
-        add_fedex_rates_button(frm);
-    },
+	refresh: function(frm) {
+		add_fedex_rates_button(frm);
+	},
 
-    get_fedex_rates: function (frm) {
-        _get_fedex_rates(frm);
-    }
+	get_fedex_rates: function(frm) {
+		_get_fedex_rates(frm);
+	}
 });
 
 
 function add_fedex_rates_button(frm) {
-    if (frm.doc.docstatus == 0) {
-        frm.add_custom_button(__('Get Fedex Rates'),
-            function () {
-                _get_fedex_rates(frm)
-            });
-    }
+	if (frm.doc.docstatus == 0) {
+		frm.add_custom_button(__('Get Fedex Rates'), function() {
+			_get_fedex_rates(frm);
+		});
+	}
 }
 
 function _get_fedex_rates(frm) {
-    $.each(frm.doc.taxes, function (index, item) {
-        if (item.account_head == "Freight and Forwarding Charges - JA") {
-            frappe.throw("Shipment Charge has already been added")
-        }
-    })
-    frappe.call({
-        freeze: 1,
-        method: 'shipment_management.api.get_rates_for_doc',
-        args: {
-            doc: frm.doc
-        },
-        callback: function (response) {
-            let options_string = ""
-            let service_dict = {}
+	frm.doc.taxes.forEach((item) => {
+		if (item.account_head == "Freight and Forwarding Charges - JA") {
+			frappe.throw("Shipment Charge has already been added");
+		}
+	})
 
-            $.each(response.message, function (index, value) {
-                let option = value.label + " - $" + value.fee + "\n"
-                options_string += option
-                service_dict[option.slice(0, -1)] = {
-                    "label": value.label,
-                    "fee": value.fee
-                }
-            })
+	frappe.call({
+		freeze: 1,
+		method: 'shipment_management.shipengine.api.get_rates',
+		args: {
+			doc: frm.doc,
+			estimate: false
+		},
+		callback: function(response) {
+			let options = [];
+			let service_dict = [];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'service_dict' is not in camel case.](https://app.codacy.com/manual/mail_67/shipment_management/pullRequest?prid=5409514)